napari/components/_viewer/view/main.py

```diff
@@ -7,17 +7,19 @@
 
 import os.path as osp
 from ....resources import resources_dir
-
+from ....util.theme import template, palettes
+palette = palettes['dark']
```

Comment:
Can we make the palette name somehow an input argument to ViewerApp and put it on the QtViewer as a property somewhere? I want to switch to a light theme in one of the examples, with something like 
```python
viewer = ViewerApp(theme='light')
viewer.add_image(img)
```
If this works it would be nice if you added an example to the examples repo, maybe `add_image_light` that was identical to the `add_image` example but with a light theme